.pipelines/android-ci.yml

```diff
@@ -17,7 +17,7 @@ trigger:
 
 pool:
   name: Azure Pipelines
-  vmImage: macOS-10.14
+  vmImage: macOS-1015
```

Comment:
nit:  have you considered this one? 'macos-latest'
I keep getting e-mail from mac os team saying mac users have to update to the latest or lose company network access.